roles/openshift-prep/tasks/prerequisites.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: "Cleaning yum repositories"
+  command: "yum clean all"
+
+- name: "Install required packages"
+  yum:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - wget
+    - git
+    - net-tools
+    - bind-utils
+    - bridge-utils
+    - bash-completion
```

Comment:
@tomassedovic no rush now but you may want to look at copying the prereq role the other providers use to be able to handle atomic